sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Amqp;
+
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    internal class AmqpMessageConstants
```

Comment:
nit:  I'll buy that the members are self-explanatory, but a comment for what these constants are used for in the wider context would be helpful for contributors.